src/Radio/RadioGroup.js

```diff
@@ -221,6 +235,15 @@ export default class RadioGroup extends Component {
         onBlur={this.handleBlur}
       >
         {Children.map(children, (child, index) => {
+          const { muiName } = child.type;
+          if (muiName === 'FormLabel') {
+            return cloneElement(child, {
+              error,
+              required,
```

Comment:
You can see me coming. We had the same discussion on the `TextField` component regarding who should have the `required` responsibility between `FormLabel ` and `RadioGroup`. It's convenient at the top level but requires to pass it down.

I would rather be coherent on that point and follow the `TextFieldLabel`.